src/theme/defaultTheme.js

```diff
@@ -69,9 +69,11 @@ const colors = {
 export const chartColors = {
   lightblue: '#00ACEB',
   darkblue: '#2b388f',
-  purple: '#A6278F',
+  purplish: '#A6278F',
```

Comment:
let's leave this as `purple` to avoid changes to existing stuff (unless we know it's not used by anything else, but I am not confident on that) :)